openproblems/tasks/chromatin_potential/__init__.py

```diff
@@ -1,7 +1,7 @@
 from ... import utils
 from . import datasets, methods, metrics, checks
 
-_task_name = "Link gene expression with chromatin accessibility"
+_task_name = "Chromatin accessibility prediction"
```

Comment:
Are you predicting gene expression from chromatin accessibility or predicting chromatin accessibility from gene expression? This implies the latter.

Reply:
We are predicting gene expression from chromatin accessibility, I'll correct this.